src/models/classes/factoryorder.js

```diff
@@ -0,0 +1,10 @@
+class FactoryOrder extends OrderCore
+{
+    constructor(products)
+    {
+        var time = GAME.model.config.orderTransportDuration;
```

Comment:
Then why does OrderCore have an `id` field? Perhaps refactor this according to what is supposed to go where?